### ISSUE TITLE
MissingMethodException: SixLabors.Fonts

### ISSUE THREAD

**Issue by hobwell** [ClosedXML/ClosedXML#2146]
## Read and complete the full issue template

Do not randomly delete sections. They are here for a reason.

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [x] Yes
- [ ] No

If you answered `No`, please test with [the latest development build](https://ci.appveyor.com/project/ClosedXML/ClosedXML/branch/develop/artifacts) first.

**Version of ClosedXML**

e.g. 0.102.0

**What is the current behavior?**

When calling worksheet.Rows().AdjustToContents() an exception is thrown:

System.MissingMethodException: 'Method not found: 'System.Collections.Generic.IEnumerable`1<SixLabors.Fonts.GlyphMetrics> SixLabors.Fonts.FontMetrics.GetGlyphMetrics(SixLabors.Fonts.Unicode.CodePoint

This appears to be related to the recent Release build of SixLabors.Fonts (2023-08-11).  Reverting to SixLabors.Fonts version 1.0.0-beta19 resolves the issue.

**What is the expected behavior or new feature?**

Row heights would be adjusted to accommodate cell text.

**Is this a regression from the previous version?**

No, it appears to be related to the latest release of SixLabors.Fonts

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

Without a code sample, it is unlikely that your issue will get attention. Don't be lazy. Do the effort and assist the developers to reproduce your problem. Code samples should be [minimal complete and verifiable](https://stackoverflow.com/help/mcve). Sample spreadsheets should be attached whenever applicable. Remove sensitive information.

**Code to reproduce problem:**
```c#
public void Main()
{
	XLWorkbook wb = new XLWorkbook();
	IXLWorksheet ws = wb.Worksheets.Add("SixLaborsFault");
	ws.Cell("A1").Value = "Short";
	ws.Cell("B1").Value = "Long";

	ws.Cell("A2").Value = "short";
	ws.Cell("B2").Value = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.";

	ws.Rows(1, 2).AdjustToContents();
}
```

- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by wagnerbsouza** [comment]
I'm getting a similar message:
```
System.MissingMethodException: 
'Method not found: 
'System.Collections.Generic.IEnumerable`1<SixLabors.Fonts.GlyphMetrics> 
SixLabors.Fonts.FontMetrics.GetGlyphMetrics(SixLabors.Fonts.Unicode.CodePoint, SixLabors.Fonts.ColorFontSupport)'.'
```

That happens when I try to `new XLWorkbook(stream)`, where `stream` is a `System.IO.Stream`.
In my case, I went from ClosedXML version 0.93.1 to 0.102.0
Apparently it "breaks" from version 0.100.3 (good) to 0.101.0 (bad).

The project I have is in .NETFramework 4.7.2, although we are planning on migrating to .NET7, which somehow might fix things with SixLabors.Fonts.

**Comment by dellos** [comment]
I got the same error message.

using Version: 101 and 102 

Project: asp.net web form, .NET Framework 4.7.2

Code:
```
  IXLWorksheet ws = wb.Worksheets.Add(dataTable,"test");
  ws.Columns().AdjustToContents();
```

I found that older versions like 75 works just fine with .NET Framework 4.7.2

**Comment by troybda** [comment]
I received the same error when trying to open an existing excel file with 102 and .NET Framework 4.7.2.  
The method GetGlyphMetrics in SixLabors.Fonts.FontMetrics version 1.0.0 was updated with a few more parameters and that is what is causing the error.

Like hobwell stated above I uninstalled ClosedXML 102 and the uninstalled SixLabors.Fonts 1.0.0 and then installed SixLabors.Fonts 1.0.0-beta19 and then installed ClosedXML 102 again and it works now.
